projects/fsastorefrontlib/src/core/my-account/store/effects/consent.effect.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { Actions, Effect, ofType } from '@ngrx/effects';
+import { Observable, of } from 'rxjs';
+import { catchError, map, switchMap } from 'rxjs/operators';
+import * as fromActions from '../actions';
+import { ConsentConnector } from '../../connectors/consent.connector';
+
+@Injectable()
+export class ConsentEffects {
+  constructor(
+    private actions$: Actions,
+    private consentConnector: ConsentConnector
+  ) {}
+
+  @Effect()
```

Comment:
Use `createEffect` instead of deprecated `@Effect`.

Reply:
Done!